diff -ur nvidia-texture-tools/src/nvimage/DirectDrawSurface.cpp org/src/nvimage/DirectDrawSurface.cpp
--- nvidia-texture-tools/src/nvimage/DirectDrawSurface.cpp	2014-02-18 20:18:40.107798300 +0800
+++ org/src/nvimage/DirectDrawSurface.cpp	2010-05-15 01:59:28.000000000 +0800
@@ -531,8 +531,7 @@
 	memset(this->reserved, 0, sizeof(this->reserved));
 
 	// Store version information on the reserved header attributes.
-	//this->reserved[9] = MAKEFOURCC('N', 'V', 'T', 'T');
-	this->reserved[9] = MAKEFOURCC('K', 'i', 'D', ' ');
+	this->reserved[9] = MAKEFOURCC('N', 'V', 'T', 'T');
 	this->reserved[10] = (2 << 16) | (0 << 8) | (8);	// major.minor.revision
 
 	this->pf.size = 32;
diff -ur nvidia-texture-tools/src/nvimage/ImageIO.cpp org/src/nvimage/ImageIO.cpp
--- nvidia-texture-tools/src/nvimage/ImageIO.cpp	2014-02-18 20:46:21.077182300 +0800
+++ org/src/nvimage/ImageIO.cpp	2010-05-15 01:59:28.000000000 +0800
@@ -14,7 +14,6 @@
 #include "FloatImage.h"
 #include "TgaFile.h"
 #include "PsdFile.h"
-#include <pngstruct.h>
 
 // Extern
 #if defined(HAVE_JPEG)
@@ -1504,4 +1503,7 @@
 		return false;
 	}
 	
-} //
\ No newline at end of file
+} // ImageIO
+
+#endif // 0
+
diff -ur nvidia-texture-tools/src/nvmath/nvmath.h org/src/nvmath/nvmath.h
--- nvidia-texture-tools/src/nvmath/nvmath.h	2014-02-18 20:25:33.931754700 +0800
+++ org/src/nvmath/nvmath.h	2010-05-15 01:59:28.000000000 +0800
@@ -85,11 +85,11 @@
 }
 
 // Replace default functions with asserting ones.
-//#define sqrt sqrt_assert
+#define sqrt sqrt_assert
 #define sqrtf sqrtf_assert
-//#define acos acos_assert
+#define acos acos_assert
 #define acosf acosf_assert
-//#define asin asin_assert
+#define asin asin_assert
 #define asinf asinf_assert
 
 #if NV_OS_WIN32
diff -ur nvidia-texture-tools/src/nvtt/CompressionOptions.cpp org/src/nvtt/CompressionOptions.cpp
--- nvidia-texture-tools/src/nvtt/CompressionOptions.cpp	2014-02-18 20:55:16.256792800 +0800
+++ org/src/nvtt/CompressionOptions.cpp	2010-05-15 01:59:28.000000000 +0800
@@ -54,11 +54,6 @@
 	m.gmask = 0x0000FF00;
 	m.rmask = 0x00FF0000;
 	m.amask = 0xFF000000;
-	/*m.rmask = 0x00000F00;
-	m.gmask = 0x000000F0;
-	m.bmask = 0x0000000F;
-	m.amask = 0x0000F000;*/
-	
 
 	m.enableColorDithering = false;
 	m.enableAlphaDithering = false;
@@ -68,10 +63,9 @@
 
 
 /// Set desired compression format.
-void CompressionOptions::setFormat(Format format, int val)
+void CompressionOptions::setFormat(Format format)
 {
 	m.format = format;
-	if (format == nvtt::Format_RGB) { m.bitcount = val; }
 }
 
 
diff -ur nvidia-texture-tools/src/nvtt/nvtt.h org/src/nvtt/nvtt.h
--- nvidia-texture-tools/src/nvtt/nvtt.h	2014-02-18 20:44:50.641009600 +0800
+++ org/src/nvtt/nvtt.h	2010-05-15 01:59:28.000000000 +0800
@@ -104,7 +104,7 @@
 		
 		NVTT_API void reset();
 		
-		NVTT_API void setFormat(Format format,int = 32);
+		NVTT_API void setFormat(Format format);
 		NVTT_API void setQuality(Quality quality);
 		NVTT_API void setColorWeights(float red, float green, float blue, float alpha = 1.0f);
 		
diff -ur nvidia-texture-tools/src/nvtt/tools/compress.cpp org/src/nvtt/tools/compress.cpp
--- nvidia-texture-tools/src/nvtt/tools/compress.cpp	2014-02-18 20:43:48.155435700 +0800
+++ org/src/nvtt/tools/compress.cpp	2010-05-15 01:59:28.000000000 +0800
@@ -144,7 +144,6 @@
 	bool silent = false;
 	bool bc1n = false;
 	nvtt::Format format = nvtt::Format_BC1;
-	int bit = 32;
 
 	const char * externalCompressor = NULL;
 
@@ -192,20 +191,9 @@
 		{
 			nocuda = true;
 		}
-		else if (strcmp("-rgb8", argv[i]) == 0)
+		else if (strcmp("-rgb", argv[i]) == 0)
 		{
 			format = nvtt::Format_RGB;
-			bit = 8;
-		}
-		else if (strcmp("-rgb16", argv[i]) == 0)
-		{
-			format = nvtt::Format_RGB;
-			bit = 16;
-		}
-		else if (strcmp("-rgb32", argv[i]) == 0)
-		{
-			format = nvtt::Format_RGB;
-			bit = 32;
 		}
 		else if (strcmp("-bc1", argv[i]) == 0)
 		{
@@ -297,9 +285,7 @@
 		printf("Compression options:\n");
 		printf("  -fast    \tFast compression.\n");
 		printf("  -nocuda  \tDo not use cuda compressor.\n");
-		printf("  -rgb8     \tRGBA 8bit format\n");
-		printf("  -rgb16     \tRGBA 16bit format\n");
-		printf("  -rgb32     \tRGBA 32bit format\n");
+		printf("  -rgb     \tRGBA format\n");
 		printf("  -bc1     \tBC1 format (DXT1)\n");
 		printf("  -bc1n    \tBC1 normal map format (DXT1nm)\n");
 		printf("  -bc1a    \tBC1 format with binary alpha (DXT1a)\n");
@@ -411,7 +397,7 @@
 	}
 
 	nvtt::CompressionOptions compressionOptions;
-	compressionOptions.setFormat(format, bit);
+	compressionOptions.setFormat(format);
 	if (fast)
 	{
 		compressionOptions.setQuality(nvtt::Quality_Fastest);
